vector/src/test/java/im/vector/app/features/settings/devices/v2/overview/SessionOverviewViewModelTest.kt

```diff
@@ -206,110 +203,6 @@ class SessionOverviewViewModelTest {
                 .finish()
     }
 
-    @Test
```

Comment:
Is there a reason to having removed all tests on `SignoutAction` ?

Reply:
I think they were lost during a merge conflict. Good catch